src/util/imageUtils.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * Determine if browser supports Webp with transparency
+ *
+ * @returns {Promise} resolves true or false
+ */
+export function checkWebpSupport() {
```

Comment:
@emuvente Any thoughts on these util functions?

Reply:
For webp they take it a bit farther verifying the dimensions after onload...
https://developers.google.com/speed/webp/faq#in_your_own_javascript